rosetta-lang/src/main/java/com/regnosys/rosetta/scoping/RosettaScopeProvider.xtend

```diff
@@ -100,32 +101,32 @@ class RosettaScopeProvider extends ImportedNamespaceAwareLocalScopeProvider {
 				}
 				case ROSETTA_FEATURE_CALL__FEATURE: {
 					if (context instanceof RosettaFeatureCall) {
-						return createExtendedFeatureScope(context.receiver, typeProvider.getRType(context.receiver))
+						return createExtendedFeatureScope(context.receiver, typeProvider.getRMetaAnnotatedType(context.receiver))
 					}
 					return IScope.NULLSCOPE
 				}
 				case ROSETTA_DEEP_FEATURE_CALL__FEATURE: {
 					if (context instanceof RosettaDeepFeatureCall) {
-						return createDeepFeatureScope(typeProvider.getRType(context.receiver))
+						return createDeepFeatureScope(typeProvider.getRMetaAnnotatedType(context.receiver).RType)
```

Comment:
This one needs support for meta features too

Reply:
Closing this as agreed no longer a requirement.